conference-finder/src/App.css

```diff
@@ -20,3 +20,27 @@
 .App-link {
   color: #09d3ac;
 }
+
+.card {
+  width: 100%;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
+}
+
+.card:hover {
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+li {
+  width: 33.33%;
```

Comment:
You should be able to change it to a certain pixel size instead of 33%. It mean's you'll be able to fit more on a bigger screen instead of just spacing them out

Reply:
I personally think the way you have done it with percentage is better for your current use case and fixing width to pixels wouldn't be good in this situation but it is down to you guys :)

Reply:
We changed it to px since it keeps the cards a consistent size which we preferred.